docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: '3.5'
+volumes:
+  node_modules:
+services:
+  app-sdk:
+    command: ./docker/run.sh
+    container_name: app-sdk
+    build:
+      context: ./
+      dockerfile: docker/Dockerfile
+    image: forgecloud/app-sdk:local
+    ports:
+      - '9080:9080'
+    environment:
+      TENANT: ${TENANT}
+      ORG_GATEWAY_URL: https://api-${TENANT}.forgeblocks.com
+      HOST: localhost
+      OAUTH_ISSUER: https://openam-${TENANT}.forgeblocks.com/openam/oauth2
+      OAUTH_KEY: ${OAUTH_KEY}
```

Comment:
I *think* this is probably OK, but want to double-check the scenario.  This value will be populated at run-time using the value of the OAUTH_KEY env var?  If so, I assume that .env is the ultimate source of the value?

If the above is correct, then this isn't creating any additional risk.  That is, it's the .env file that's potentially exposing the secret, not this file...

Reply:
that's correct, docker compose picks up values from `.env` - https://docs.docker.com/compose/env-file/